ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -590,3 +591,25 @@ def get_all_storageclass():
         )
     ]
     return storageclass
+
+
+def setup_ceph_debug():
+    """
+    Set Ceph to run in debug log level using a ConfigMap.
+    This functionality is available starting OCS 4.7.
+
+    """
+    ceph_debug_log_configmap_data = templating.load_yaml(
+        constants.CEPH_CONFIG_DEBUG_LOG_LEVEL_CONFIGMAP
+    )
+    ceph_debug_log_configmap_data["data"][
+        "config"
+    ] = f"|\n{constants.ROOK_CEPH_CONFIG_VALUES}{constants.CEPH_DEBUG_CONFIG_VALUES}"
```

Comment:
We don't need to tweak the string to conform yaml quoting, it will be done automatically. Just se the value to the string we expect there:
```
ceph_debug_log_configmap_data["data"]["config"] = constants.ROOK_CEPH_CONFIG_VALUES + constants.CEPH_DEBUG_CONFIG_VALUES
```

Reply:
thanks Martin, will do